From 25c0894d9d06f61310e26e46d4b1f92eb00a3b42 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Sun, 24 Jul 2022 20:52:36 -0400
Subject: [PATCH] part_efi: Avoid deluge of print when device is not GPT

---
 disk/part_efi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/disk/part_efi.c b/disk/part_efi.c
index 27f3e4dda31..7798049472f 100644
--- a/disk/part_efi.c
+++ b/disk/part_efi.c
@@ -82,7 +82,7 @@ static int validate_gpt_header(gpt_header *gpt_h, lbaint_t lba,
 	/* Check the GPT header signature */
 	if (le64_to_cpu(gpt_h->signature) != GPT_HEADER_SIGNATURE) {
 		if (le64_to_cpu(gpt_h->signature) != 0)
-			printf("%s signature is wrong: 0x%llX != 0x%llX\n",
+			debug("%s signature is wrong: 0x%llX != 0x%llX\n",
 			       "GUID Partition Table Header",
 			       le64_to_cpu(gpt_h->signature),
 			       GPT_HEADER_SIGNATURE);
-- 
2.35.1

